src/service-broker.js

```diff
@@ -964,7 +964,18 @@ class ServiceBroker {
 	 */
 	registerInternalServices(opts) {
 		opts = utils.isObject(opts) ? opts : {};
-		this.createService(require("./internals")(this), opts["$node"]);
+		const interalsSchema = require("./internals")(this);
+		let definitiveSchema = {};
+		// If it's present any custom definition, define it as the root schema and the default one as a mixin
+		if (opts["$node"]) {
+			definitiveSchema = opts["$node"];
+			if (!definitiveSchema.mixins) definitiveSchema.mixins = [];
+			definitiveSchema.mixins.push(interalsSchema);
+		// Otherwise, just use the default one
```

Comment:
this comment produce a style issue (maybe remove it ? ) : https://app.codacy.com/gh/moleculerjs/moleculer/pullRequest?prid=10738424

Reply:
It's just a style issue, I aligned it to the ``else`` level as it was related to that. I'm going to fix it

Reply:
Yes, I know why you aligned it like that :) . But it add an eslint error :/

Reply:
Just added the fix 👍